docker/Dockerfile

```
@@ -1,51 +1,22 @@
-# ----------------
-# STEP 1:
-FROM node:erbium-alpine AS build-z2m
+FROM node:12.20.1-buster
```

```suggestion
FROM node:12.20.1-buster AS build-z2m
```

BTW it would prefer to stick to alpine if possibile

---

Yeah. Though you won't get the static binaries. Not the end of the world if the steps are optimised and cached